core/new-gui/src/app/workspace/component/result-panel/result-panel.component.html

```diff
@@ -1,5 +1,33 @@
+<ng-template #content let-c="close" let-d="dismiss">
+  <div class="modal-header">
+    <h4 class="modal-title">Row Details</h4>
+    <button type="button" class="close" aria-label="Close" (click)="!d('Cross click')">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <pre>{{currentDisplayRow}}</pre>
+  </div>
+  <div class="modal-footer">
+    <button type="button" class="btn btn-outline-dark" (click)="!c('Close click')">Close</button>
+  </div>
+</ng-template>
+
+
 <div class="texera-workspace-result-panel-body texera-workflow-component-body">
-  <p>
-    result-panel works!
-  </p>
+  <div *ngIf="showMessage">
+    <p>{{ message }}</p>
+  </div>
+  <div class="result-table" *ngIf="currentColumns && !showMessage">
+    <mat-table #table [dataSource]="currentDataSource">
+      <ng-container *ngFor="let column of currentColumns" matColumnDef={{column.columnDef}}>
+        <mat-header-cell *matHeaderCellDef>{{ column.header }}</mat-header-cell>
+        <mat-cell  *matCellDef="let row">{{ column.cell(row) }}</mat-cell>
+      </ng-container>
+      <mat-header-row *matHeaderRowDef="currentDisplayColumns"></mat-header-row>
+      <mat-row class="result-table-row" *matRowDef="let row; columns: currentDisplayColumns;"
+          (click)="getRowDetails(row, content)"></mat-row>
```

Comment:
Where does this content in (row, content) come from? I don't see it being initialized anywhere above.

Reply:
`content` is defined at the line `<ng-template #content let-c="close" let-d="dismiss">`.
This part of the code is straightly provided by the ng-bootstrap library.

https://ng-bootstrap.github.io/#/components/modal/examples